modules/manage/pages/kubernetes/troubleshooting/k-troubleshoot.adoc

```diff
@@ -406,6 +406,82 @@ kubectl logs -f <pod-name> --namespace <namespace>
 You can use the output to debug your deployment.
 //end::deployment-pod-pending[]
 
+//tag::deployment-failed-scheduling[]
+=== FailedScheduling
+
+If you see this error, your cluster does not have enough nodes to satisfy the anti-affinity rules:
+
+[.no-copy]
+----
+Warning  FailedScheduling  18m  default-scheduler  0/1 nodes are available: 1 node(s) didn't match pod anti-affinity rules. preemption: 0/1 nodes are available: 1 No preemption victims found for incoming pod..
+----
+
+The Helm chart configures default `podAntiAffinity` rules to make sure that only one Pod running a Redpanda broker is scheduled on each worker node. To learn why, see xref:deploy:deployment-option/self-hosted/kubernetes/k-requirements.adoc#number-of-workers[Number of workers].
+
+To resolve this issue, do one of the following:
+
+- Create additional worker nodes.
+
+- Modify the anti-affinity rules (for development purposes only).
++
+If adding nodes is not an option, you can modify the `podAntiAffinity` rules in your StatefulSet to be less strict.
++
+[tabs]
+======
+Helm + Operator::
++
+--
+.`redpanda-cluster.yaml`
+[,yaml]
+----
+apiVersion: cluster.redpanda.com/v1alpha1
+kind: Redpanda
+metadata:
+  name: redpanda
+spec:
+  chartRef: {}
+  clusterSpec:
+    statefulset:
+      podAntiAffinity:
+        type: soft
```

Comment:
I set this to type=custom which worked for me (based off of a kapa suggestion). Is soft the preferable way?

Reply:
Soft still allows scheduling the Pods onto separate workers but only if they’re available. The rule becomes [`preferredDuringSchedulingIgnoredDuringExecution`](https://github.com/redpanda-data/helm-charts/blob/5f287d45a3bda2763896840e505fb3de82b968b6/charts/redpanda/templates/_statefulset.tpl#L88)

Reply:
@chrisseto or @RafalKorepta can confirm

Reply:
Type `custom` is a way to provide your own pod anti affinity. 

https://github.com/redpanda-data/helm-charts/blob/5f287d45a3bda2763896840e505fb3de82b968b6/charts/redpanda/templates/_statefulset.tpl#L81-L100